src/webgpu/shader/execution/expression/call/builtin/textureLoad.spec.ts

```diff
@@ -483,7 +483,12 @@ Parameters:
       size,
       usage: GPUTextureUsage.COPY_DST,
     };
+
```

Comment:
If this was changed to

```js
.beforeAllSubcases(t => t.skipIf(typeof VideoFrame === 'undefined'))
```

it would get skipped before creating a device.